tap_zendesk/schemas/ticket_metric_events.json

```diff
@@ -0,0 +1,101 @@
+{
+  "properties": {
+    "id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "instance_id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "metric": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "ticket_id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "time": {
+      "type": [
+        "null",
+        "string"
+      ],
+      "format": "date-time"
+    },
+    "type": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "sla": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "properties": {
+        "target": {
+          "type": [
+            "integer"
+          ]
+        },
+        "business_hours": {
+          "type": [
+            "boolean"
+          ]
+        },
+        "policy": {
+          "type": [
+            "object"
+          ],
+          "properties": {
+            "id": {
+              "type": [
+                "integer"
+              ]
+            },
+            "title": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "description": {
+              "type": [
+                "null",
+                "string"
+              ]
+            }
+          }
+        }
+      }
+    },
```

Comment:
This is the portion of data that this PR is for. This will allow us to know which policy applies to which ticket.